### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is empty for the "Twitter", "Facebook" and "Share" links

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#56]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/items?itemName=donjayamanne.python in EDGE
2. Navigate to the "Twitter", "Facebook" and "Share" links available under "More Info" header in the RHS of the page
3. Observe the "Name" property in Inspect

**Actual Result:**
The "Name property is empty
**_Narrator Behaviour:_**
Narrator announces as "Link"

**Expected Result:**
The "Name" property should be specific
Eg: "Twitter link"
      "Facebook Link"
      "Share Link"

**User Impact:**
Visually impaired users  will not get to know the purpose of the links as the screen reader will not announce anything when the user arrives at the installation command
Providing the name property enables the compatibility with the Screen Reader, which may be used by people with disabilites.

**Screenshot:**
![name property is empty for twitter link](https://cloud.githubusercontent.com/assets/24825441/21758840/98b53558-d665-11e6-8722-cb74f81ae852.png)

**Comment by waderyan** [comment]
Marketplace issue.
